components/server/src/auth/rate-limiter.ts

```diff
@@ -85,6 +85,7 @@ const defaultFunctions: FunctionsConfig = {
     getOpenPorts: { group: "default", points: 1 },
     openPort: { group: "default", points: 1 },
     closePort: { group: "default", points: 1 },
+    updateRepoStatus: { group: "default", points: 1 },
```

Comment:
I'm not sure about the value of rate limit, but we can know it from grafana if there is too many RESOURCE_EXHAUSTED

Reply:
We are going via feature flags and can monitor what is happening. We already have guards in place though which should prevent sending a request or updating DB.